### PR TITLE
Create participating.md

### PR DESCRIPTION
**Pull request by jepler**

### CODE REVIEW

File: participating.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+Anyone is welcome to participate in the CircuitPython weekly meeting. Please
+read our [Code of Conduct](https://github.com/adafruit-circuitpython-weekly-meeting/blob/master/CODE_OF_CONDUCT.md)
+before participating to help this project stay welcoming.
+
+The meeting is held in the English language and usually lasts between 60 and 90
+minutes.  If you can only attend part of the meeting, that's absolutely fine.
+
+To participate in the meeting, or just attend it live as a listener, join us on
+the [Adafruit Discord](https://adafru.it/discord). There is a _#CircuitPython_
+text channel and a _#CircuitPython_ voice channel.
```
Comment (kattni) [inline participating.md:12]: The channel name is not camel cased. It should be _#circuitpython_.

File: participating.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+Before speaking in the meeting, ask any Community Moderator to add you to the
+_@CircuitPythonistas_ role. In this role, you will receive a small number of
+notifications each week, almost always related to the meeting. You can mute
+these notifications or request to leave the role anytime you like.
+
+While we love to hear the voices of everyone in the community, we also
+understand that it doesn't always work out to speak during the
+meeting—whether you don't have a microphone, are in a noisy environment, are
+simply uncomfortable speaking, or for any other reason—so the moderator will
```
Comment (kattni) [inline participating.md:26]: Consider using "host" instead of "moderator". Same applies for all other uses of "moderator", with the obvious exception of the "Community Moderator" reference.

File: participating.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+read those notes aloud 
+
+We encourage you to listen to a meeting or two before participating, so that
+you know better what to expect. You can watch old meetings [on youtube](https://www.youtube.com/watch?v=lvo-seDO4yk&list=PLjF7R1fz_OOUvw7tMv45xjWp0ht8yNgg0)
+or listen to them on podcast services. 
+
+# Parts of the meeting
```
Comment (kattni) [inline participating.md:33]: This seems awkward. Perhaps "Meeting Sections"?

File: participating.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+# Before the meeting
+
+Remember to [check the meeting calendar](https://open-web-calendar.herokuapp.com/calendar.html?url=https%3A%2F%2Fraw.githubusercontent.com%2Fadafruit%2Fadafruit-circuitpython-weekly-meeting%2Fmaster%2Fmeeting.ical&title=CicuitPython%20Weekly%20Meeting%20Schedule&tab=agenda&tabs=month&tabs=agenda).
+The meeting is usually every Monday at 1PM [US Eastern Time (UTC-0400 or UTC-0500)](https://en.wikipedia.org/wiki/Eastern_Time_Zone),
+but we sometimes change the schedule for US holidays and other special events.
+If you're outside the US remember that the dates for the start and end of your
+"summer" time may differ from the rule followed in the US.
+
+Each meeting is accompanied by a notes document in google docs. You can find
```
Comment (kattni) [inline participating.md:92]: "Google Docs" should be capitalised.

File: participating.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+paragraphs) but if the moderator is going to be reading them make sure they
+have enough context that what they say will make sense.
+
+# During the meeting
+
+Join a few minutes before the meeting if you want to check that your audio is working.
+
+Please give the moderator a few minutes after the start of the hour to arrive.
+The Discord meeting starts after an internal Adafruit meeting which often goes
+slightly over its alloted time.
```
Comment (kattni) [inline participating.md:123]: I think it's fine to simply say "The Discord meeting starts after an internal Adafruit meeting." If you wish to keep the full sentence, "allotted" has two t's as well.

**Review by kattni — CHANGES_REQUESTED** [review]
This is great! Thank you for putting it together. I have a few suggestions below.

File: participating.md (hunk 2 above)
Comment (jepler) [reply participating.md:26]: would you believe I could not remember the word "host"?

Comment (kattni) [reply participating.md]: Entirely! 😂

File: participating.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+understand that it doesn't always work out to speak during the
+meeting—whether you don't have a microphone, are in a noisy environment, are
+simply uncomfortable speaking, or for any other reason—so the host will
+read those notes aloud 
+
+We encourage you to listen to a meeting or two before participating, so that
+you know better what to expect. You can watch old meetings [on youtube](https://www.youtube.com/watch?v=lvo-seDO4yk&list=PLjF7R1fz_OOUvw7tMv45xjWp0ht8yNgg0)
+or listen to them on podcast services. 
+
+# Meeting Section
```
Comment (kattni) [inline participating.md:33]: Probably "Secctions" is better as it references multiple.

File: participating.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+To participate in the meeting, or just attend it live as a listener, join us on
+the [Adafruit Discord](https://adafru.it/discord). There is a _#circuitpython_
+text channel and a _#circuitpython_ voice channel.
+
+The text from the text channel and the audio from the voice channel are
+recorded and uploaded to YouTube and podcast services. Please keep this in
+mind if you plan to participate.
+
+Before speaking in the meeting, ask any Community Moderator to add you to the
+_@CircuitPythonistas_ role. In this role, you will receive a small number of
```
Comment (kattni) [inline participating.md:19]: It occurs to me that we should reword this to indicate that being added to the role is **required** for speaking. Note that the role is also not camel cased.

```suggestion
If you wish to speak in the meeting, you must ask any Community Moderator to add you to the
_@circuitpythonistas_ role. In this role, you will receive a small number of
```

File: participating.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+notifications each week, almost always related to the meeting. You can mute
+these notifications or request to leave the role anytime you like.
+
+While we love to hear the voices of everyone in the community, we also
+understand that it doesn't always work out to speak during the
+meeting—whether you don't have a microphone, are in a noisy environment, are
+simply uncomfortable speaking, or for any other reason—so the host will
+read those notes aloud
```
Comment (kattni) [inline participating.md:27]: Missing period.

File: participating.md [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+Please give the host a few minutes after the start of the hour to arrive.
+The Discord meeting starts immediately after an internal Adafruit meeting.
+
+When not actively speaking, please mute your microphone.
+
+In the sections where you will participate, be attentive to the order of the
+meeting so that you're ready to un-mute and speak when called on.
+
+If the host accidentally skips you, place a note in the text chat to make
+them aware of it.
```
Comment (kattni) [inline participating.md:130]: Consider rewording this a bit. It might be a little unclear as is. My wording might not be the best either, but it's an idea.

```suggestion
If the host accidentally skips you,  please send a note in the Discord text chat to make
them aware of it.
```

File: participating.md [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+
+## In The Weeds
+
+In The Weeds topics are discussed in the order they appear in the notes document.
+
+If you've brought an In The Weeds topic, you will introduce it and then anyone
```
Comment (kattni) [inline participating.md:136]: I think clarify here:
```suggestion
If you've brought an In The Weeds topic, the host will introduce you, and you may introduce the topic. Then anyone
```

File: participating.md [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+who is interested in the discussion can unmute and speak freely, or participate
+in the text chat as they prefer. At the end of the discussion, anyone who is
+going to take action will state so clearly, such as "I will add additional
+information to the relevant Issue", "I will fix the bug using the approach we
+selected", etc. The person who brought the In The Weeds topic will also
+briefly summarise the discussion in the notes document. Please do this
+promptly, as the host has to finalize the notes document and upload it to
+github shortly after the meeting concludes.
```
Comment (kattni) [inline participating.md:144]:
```suggestion
GitHub shortly after the meeting concludes.
```

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks for the fixes! Found a few more things. Of course, right?

**Review by kattni — APPROVED** [review]
This is good to go! We can iterate over time as needed.
